### PR TITLE
Create 2019-03-04-prototype-release-v4.md

### PR DESCRIPTION
**Pull request by darkdrgn2k**

### CODE REVIEW

File: _timeline/2019-03-04-prototype-release-v4.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Prototype Release v4
```
Comment (makew0rld) [inline _timeline/2019-03-04-prototype-release-v4.md]: v0.4?

**Review by makew0rld — APPROVED** [review]

**Review by benhylau — COMMENTED** [review]
Shall we mention key features like Profiles and Yggdrasil instead of the more generic "new modules, features, and bug fixes"?

**Comment by benhylau** [comment]
To someone looking at our website, I think features like Configuration Profiles and Status Web Page would be relatable?

File: _timeline/2019-03-04-prototype-release-v4.md (hunk 1 above)
Comment (benhylau) [reply _timeline/2019-03-04-prototype-release-v4.md]: @darkdrgn2k ^

**Comment by dasanchez** [comment]
@darkdrgn2k @Shrinks99 @garrying @benhylau 
The readme for prototype-cjdns-pi has a broken link on "Frequently Asked Questions".

When it gets embedded in tomesh.net, this readme gets funny:
- The MODULES.md link breaks 
- The Hardware Table loses its formatting
I'm inclined to remove the page from the website and link directly to the [GitHub page](https://github.com/tomeshnet/prototype-cjdns-pi) straight from the 'Build a Node' option.

Thoughts?

File: _timeline/2019-03-04-prototype-release-v4.md (hunk 1 above)
Comment (dasanchez) [reply _timeline/2019-03-04-prototype-release-v4.md]: I changed it to v0.4

**Comment by makew0rld** [comment]
@dasanchez 
> The MODULES.md link breaks

That was me. The link is `[MODULES.md](./docs/MODULES.md)`, as before links on the README were branch dependent, linking to something like https://github.com/tomeshnet/prototype-cjdns-pi/blob/master/docs/MODULES.md . That way on the develop branch it still links to the develop version of the file. I never thought about how it would affect embedding. Should it and other similar links be changed?

**Comment by dasanchez** [comment]
@makeworld-the-better-one 
I [created an issue](https://github.com/tomeshnet/prototype-cjdns-pi/issues/351) in the prototype-cjdns-pi repo to address the broken links in the README file. It might be something as silly as losing the dot before the `docs` folder, but it's not a deal breaker for the purposes of this pull request.

**Review by dasanchez — APPROVED** [review]

**Pull request merged**